packages/state-transition/src/util/computeAnchorCheckpoint.ts

```diff
@@ -0,0 +1,38 @@
+import {ChainForkConfig} from "@lodestar/config";
+import {ssz, phase0} from "@lodestar/types";
+import {GENESIS_SLOT, ZERO_HASH} from "@lodestar/params";
+import {BeaconStateAllForks} from "../types.js";
+import {blockToHeader} from "./blockRoot.js";
+import {computeCheckpointEpochAtStateSlot} from "./epoch.js";
+
+export function computeAnchorCheckpoint(
```

Comment:
looks like this is copied from `packages/beacon-node/src/chain/initState.ts`
need to deduplicate it